cypress/support/commands/agenda-commands.js

```diff
@@ -493,9 +498,12 @@ const approveAndCloseDesignAgenda = () => {
       timeout: 12000,
     });
 
-  cy.waitUntil(() => cy.get('.vl-loader').should('not.be.visible'), {
-    verbose: true, timeout: 60000,
-  });
+  cy.get('.vl-loader', {
+    timeout: 60000,
+  }).should('not.exist');
+  // cy.waitUntil(() => cy.get('.vl-loader').should('not.be.visible'), {
```

Comment:
Voor mij part mogen die unused comments weg ze. tenzij je ze er voor historische redenen wil bij houden?

Reply:
Die waitUntil wordt niet gebruikt zoals omschreven in de documentatie ervan, die timeout van 60 seconden werkte dus niet.
Zal de commentaar verwijderen